foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/sessions/PropertiesHandler.java

```diff
@@ -564,10 +564,15 @@ protected static class TargetDatabaseProp extends Prop {
             valueArray = new Object[][] {
                 {TargetDatabase.Auto, pcg + "DatabasePlatform"},
                 {TargetDatabase.Oracle, pcg + "OraclePlatform"},
-                {TargetDatabase.Oracle8, pcg + "oracle.Oracle8Platform"},
-                {TargetDatabase.Oracle9, pcg + "oracle.Oracle9Platform"},
-                {TargetDatabase.Oracle10, pcg + "oracle.Oracle10Platform"},
-                {TargetDatabase.Oracle11, pcg + "oracle.Oracle11Platform"},
+                {TargetDatabase.Oracle8, pcg + "Oracle8Platform"},
+                {TargetDatabase.Oracle9, pcg + "Oracle9Platform"},
+                {TargetDatabase.Oracle10, pcg + "Oracle10Platform"},
+                {TargetDatabase.Oracle11, pcg + "Oracle11Platform"},
+                {TargetDatabase.Oracle12, pcg + "Oracle12Platform"},
+                {TargetDatabase.Oracle18, pcg + "Oracle18Platform"},
+                {TargetDatabase.Oracle19, pcg + "Oracle19Platform"},
+                {TargetDatabase.Oracle21, pcg + "Oracle21Platform"},
```

Comment:
Is it any reason, why there are changes `pcg + "oracle.Oracle11Platform"` -> `pcg + "Oracle11Platform"`? I mean `oracle` package remove?

Reply:
No, restored original package. Also this change was moved into a separate PR #2479